examples/tv-casting-app/tv-casting-common/src/TargetVideoPlayerInfo.cpp

```diff
@@ -40,7 +40,10 @@ CHIP_ERROR TargetVideoPlayerInfo::Initialize(NodeId nodeId, FabricIndex fabricIn
         mIpAddress[i] = ipAddress[i];
     }
 
-    chip::Platform::CopyString(mDeviceName, chip::Dnssd::kMaxDeviceNameLen + 1, deviceName);
+    if (deviceName)
+    {
+        chip::Platform::CopyString(mDeviceName, chip::Dnssd::kMaxDeviceNameLen + 1, deviceName);
```

Comment:
mDeviceName is already `TargetVideoPlayerInfo.h:    char mDeviceName[chip::Dnssd::kMaxDeviceNameLen + 1]` so CopyString should be a be to auto-deduce size. Could we use `chip::Platform::CopyString(mDeviceName, deviceName)` ?

Reply:
addressed